CHANGELOG.md

```diff
@@ -10,6 +10,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ### Added
 
 ### Changed
+* `crud.select()` without conditions starts with an `after` key if
```

Comment:
Starts "with an `after` key" or starts "after `after` key"? Is starting after `after` key is added to conditions as usual?

Reply:
I rewrote the message, but I don't mind to delete it.

Reply:
If it's an optimization, I think it's better to write something like "optimize `crud.select({}, {after = tuple})`" without digging too deep on what exactly was optimized. However, to prove that it's really an optimization we need to run some performance tests after and before the fix.

If it isn't something that users will notice even in terms of optimization, it's better to remove changelog entry at all to not confuse them.

Reply:
It seems to me that this is not a very common case of using a crud.select to write and mantain a separate performance test. So i've just remove the entry from the changelog.

Reply:
But you can see the difference with the code:

```lua
box.schema.create_space('developers', {
    format = {
        {name = 'id', type = 'unsigned'},
    }
})
box.space.developers:create_index('primary_index', {
    parts = {
        {field = 1, type = 'unsigned'},
    },
})
-- Fill the space.
for i=1, 1000000 do
     box.space.developers:insert({i})
end
crud.select('developers', {}, {after = {999990}})
```

Reply:
> It seems to me that this is not a very common case of using a crud.select to write and maintain a separate performance test.

It's quite the contrary: most performance tests in crud module measure `select` performance. But it's okay if you don't want to write and maintain a performance test: it's really bothersome. Moreover, it's not like it will be widely used after this patch.

Reply:
I've tried to run benchmark tests, but dispersion is too high to really catch the performance upgrade.

I think it's still better to leave a changelog entry even if we couldn't specify what actual improvement is in numbers.

Reply:
Thank you, I've added:
```patch
--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -10,6 +10,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ### Added
 
 ### Changed
+* Optimize `crud.select()` without conditions and with `after`.
 
```